README.md

```diff
@@ -99,6 +102,99 @@ The server can be configured via the following command line options or environme
 |`CHANNEL_NAME`         | `--channel-name`       | Redis channel name to be used for communication between the server and client. | `hubot-huntflow-reloaded`                 |
 |`TZ`                   |                        | Timezone for for scheduler **(for Docker container only)**.                    | Europe/Moscow                             |
 
+### How to run server for development
+
+1. Create virtual environment and install required packages in it.
+    ```bash
+    virtualenv -p python3 hubot-huntflow-reloaded-env
+    source hubot-huntflow-reloaded-env/bin/activate
+    cd hubot-huntflow-reloaded
+    pip install -r server/requirements.txt
+    ```
+
+2. To start work with huntflow-reloaded server you need to setup the PostgreSQL database and apply the migrations. 
+    The easiest way to do it is to specify the required param `POSTGRES_PASSWORD`, run
+    ```bash
+    cd server/docker/
+    docker-compose up
+    ```
+    and then stop it by pressing Ctrl-C. It automatically prepares the database to be used.
+    
+    Notice, that `docker-compose` runs three containers and if something goes wrong you need to look at `huntflow-reloaded-server` logs first of all to get to the bottom of the problem.
+    
+3. Deploy Rocket.Chat server locally and run the Hubot. For the details see Rocket.Chat [README](https://github.com/tolstoyevsky/mmb/tree/master/rocketchat), Rocket.Chat Hubot adapter [README](https://github.com/tolstoyevsky/mmb/tree/master/hubot-rocketchat) and [client configuration](#configuration) section.
+    The Hubot runs also the Redis server. If you don't need the client manually run the Redis server
+    
+    ```bash
+    docker-compose up redis
+    ```
+    Also to debug you can subscribe to the Redis channel to see the reminders as client would receive. 
+    Find the name of the Redis container in output of the command,
+    ```
+    docker ps
+    ```
+    start the bash session in the running container via
+    ```
+    docker exec -it container_name /bin/bash/
+    ```
+     and subscribe to the Redis channel.
+    ```bash
+    redis-cli -h 127.0.0.1 -p 16379 
+    SUBSCRIBE hubot-huntflow-reloaded
+    ```
+    Notice, that if you change the default settings you need to use them.
+
+4. Run PostgreSQL server
+    ```bash
+    docker-compose up postgres-hf
+    ```
+
+5. Run huntflow-reloaded-server
+    ```bash
+    env PYTHONPATH=$(pwd)/server ../hubot-huntflow-reloaded-env/bin/python3 server/bin/server.py --redis-port=16379
```

Comment:
I you have already activated the virtual environment, you don't need to use `../hubot-huntflow-reloaded-env/bin/`
or
simply remove `source hubot-huntflow-reloaded-env/bin/activate`.